Utils/RequestDataFormatter.php

```diff
@@ -6,23 +6,31 @@
 
 final class RequestDataFormatter
 {
-    const SHOPWARE_KEYS = ['module' => null, 'controller' => null, 'action' => null];
-
-    public static function forRedirect(array $data): array
-    {
-        if (!$data) {
-            return [];
-        }
-
-        return array_diff_key($data, self::SHOPWARE_KEYS);
-    }
+    const ALLOWED_PAYMENT_DETAIL_V67_KEYS = [
+        'MD' => null,
+        'PaReq' => null,
+        'PaRes' => null,
```

Comment:
In case the transparent redirect is removed then only the GET redirect 3DS1 can be used (which is the default behaviour in the V67 API) with the `redirectResult` field in that case these fields could be removed since none of these will be used anymore.

Maybe also a good idea to add a not to the release notes to check with tech support if their Merchant account is using the default behaviour and no extra Account Data Property is set for the 3DS1 redirect http method.

```suggestion
```

Reply:
@cyattilakiss, @acampos1916 

Transparent redirect is indeed removed.

I have kept the fields to be inline with the API docs: 
https://docs.adyen.com/api-explorer/#/CheckoutService/v67/post/payments/details
Where MD,  PaReq, PaRes are still allowed.

**Regarding testing:**
The debug logs contain the `redirectResult` parameter used in the request to payments/details.
For example on PayPal redirect payments.